domains/ki.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Anandkrishna34",
+        "email": "aanandkrishna344@gmail.com"
+    },
+    "record": {
+        "A": ["69.30.249.53"]
```

Comment:
i cannot reach your website
"502 Bad Gateway"

Reply:
it is a dbh proxy